src/prefect/infrastructure/docker.py

```diff
@@ -288,6 +312,22 @@ def preview(self):
         finally:
             docker_client.close()
 
+    def _get_infrastructure_pid(self, container_id: str) -> str:
+        """Generates a docker infrastructure_pid string in the form of
+        `<docker_host_base_url>:<container_id>`.
+        """
+        docker_client = self._get_client()
+        base_url = docker_client.api.base_url
```

Comment:
Do Docker client urls ever have secrets in them?

Reply:
Ooph. I have no clue. I'll see if somebody on the platform team might have a better idea.

Reply:
tldr; probably not.